cmd/state-svc/service.go

```diff
@@ -82,3 +83,26 @@ func (s *service) Wait() error {
 	}
 	return nil
 }
+
+func (s *service) RunIfNotAuthority(checkWait time.Duration, ipComm svcctl.IPCommunicator, fn func(err error)) {
+	addr := portText(s.server)
+
+	go func() {
+		select {
+		case <-s.ctx.Done():
+			return
+		case <-time.After(checkWait):
+			checkedAddr, err := svcctl.LocateHTTP(ipComm)
+			if err == nil && (addr == "" || checkedAddr != addr) {
```

Comment:
```suggestion
			if err == nil && checkedAddr != addr {
```

`addr == ""` is pointless noise considering `portText()` would at minimum return `:`.

Reply:
This concern is a great catch, I hope you don't mind that I solved it slightly differently by permitting the portText function to return empty string in some reasonable conditions.

Reply:
Long as it's solving a real problem it sounds good to me 👍